src/Components/Medicine/CreatePrescriptionForm.tsx

```diff
@@ -69,17 +90,38 @@ export default function CreatePrescriptionForm(props: {
               optionLabel={(key) => t("PRESCRIPTION_ROUTE_" + key)}
               optionValue={(key) => key}
             />
-            <NumericWithUnitsFormField
-              className="flex-1"
-              label={t("dosage")}
-              {...field("dosage", RequiredFieldValidator())}
-              required
-              units={["mg", "g", "ml", "drop(s)", "ampule(s)", "tsp"]}
-              min={0}
-            />
+            {field("dosage_type").value === "TITRATED" ? (
+              <div className="flex w-full gap-4">
+                <NumericWithUnitsFormField
+                  className="flex-1"
+                  label="Start Dosage"
+                  {...field("base_dosage", RequiredFieldValidator())}
+                  required
+                  units={["mg", "g", "ml", "drop(s)", "ampule(s)", "tsp"]}
+                  min={0}
+                />
```

Comment:
This component seems to be duplicated in 4 places as of now. Consider refactoring it to be a reusable component, so that later on if we intent to modify the available units, we can do it in once place without worrying if it has been reflected in all existing places.